### PR TITLE
Microsoft Windows AMSI Mapping

### PR DESCRIPTION
**Pull request by eepstain**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by nkanon — COMMENTED** [review]
Please see my comments 

Thanks!

File: Packs/MicrosoftWindowsAMSI/ModelingRules/MicrosoftWindowsAMSI/MicrosoftWindowsAMSI_schema.json
Comment (nkanon) [inline Packs/MicrosoftWindowsAMSI/ModelingRules/MicrosoftWindowsAMSI/MicrosoftWindowsAMSI_schema.json:1]: Please add log_level to the schema file

File: Packs/MicrosoftWindowsAMSI/README.md
Comment (nkanon) [inline Packs/MicrosoftWindowsAMSI/README.md:1]: Should we use XSIAM tags?

**Review by nkanon — APPROVED** [review]

File: Packs/MicrosoftWindowsAMSI/ModelingRules/MicrosoftWindowsAMSI/MicrosoftWindowsAMSI_schema.json
Comment (eepstain) [reply Packs/MicrosoftWindowsAMSI/ModelingRules/MicrosoftWindowsAMSI/MicrosoftWindowsAMSI_schema.json:1]: Added to schema, thank you!

**Pull request merged**